technical-docs/designs/repos-merge.md

```diff
@@ -0,0 +1,284 @@
+  and verify all pipelines continue to work.
+- Move `guides\contributing.md` file to `docs\contribution` folder
+- Move `guides` folder to `docs` folder
+- Move `technical-docs\designs` to `docs\contribution\designs` folder
+- Remove `.github\workflows` folder since it is not being used
+
+#### Testing Changes
+
+After making the above changes, need to make sure that the follwing tests pass.
+
+- Verify cli build pipeline is working
+- Verify cli integration test pipeline is working
+- Verify cli release pipeline is working
+- Verify cli autogenerated docs are working on GitHub pages
+- Verify all docs links are working
+
+#### SPK repo after changes
+
+`spk` repo would look like below after implementing the above changes.
+
+```
+├── build
+    ├── pipelines
+        ├── templates
+        ├── azure-pipelines.yml
+        ├── release-pipeline.yml
+        ├── smoke-test-pipeline.yml
+     ├── patches
+        ├── 001-azure-devops-node.patch
+     ├── publishing
+     ├── tools
+        ├── generateDoc.ts
+        ├── locateAliases.ts
+        ├── release-version-bump.sh
+        ├── tag-release.sh
+        ├── update_introspection.sh
+├── docs
+    ├── commands
+    ├── contribution
+        ├── contributing.md
+        ├── designs
+    ├── guides
+├── src
+    ├── lib
+    ├── commands
+    ├── logger
+├── tests
+├── typings
+    ├── ssh-url
+├── .editorconfig
+├── .gitignore
+├── jest.config.js
+├── tsconfig.json
+├── tslint.json
+├── CHANGELOG.md
+├── README.md
+├── package.json
+├── webpack.config.js
+├── .yarn.lock
+```
+
+## Phase 2: Move SPK to Bedrock Repo (under review)
+
+Moving SPK repo to Bedrock involves following changes.
+
+### Activities
+
+- Update `.readme.md` in `bedrock` repo with contents from `spk` repo as
+  appropriate. Please note this is not a complete merge since it is a new
+  landing page of combined repos.
+- Create or update `changelog.md` file. (_TBD_)
+- Move all files and folder from the root of `spk` to `bedrock` root
+
+### Combined repo
+
+The combined `bedrock` repo would look like below after implementing the above
+changes.
+
+_color legend:_
+
+```diff
+# - no change from the merge process
++ - all files and folders including sub folders came from `spk` repo
+! - changed before merge in `bedrock` repo
+```
+
+```diff
+# ├── .github
+# ├── gitops
+! ├── cluster
++ ├── build
++     ├── pipelines
++         ├── templates
++         ├── azure-pipelines.yml
++         ├── release-pipeline.yml
++         ├── smoke-test-pipeline.yml
++     ├── patches
++         ├── 001-azure-devops-node.patch
++     ├── publishing
++     ├── tools
++         ├── generateDoc.ts
++         ├── locateAliases.ts
++         ├── release-version-bump.sh
++         ├── tag-release.sh
++         ├── update_introspection.sh
++ ├── docs
++     ├── commands
++     ├── contribution
++         ├── contributing.md
++         ├── designs
++     ├── guides
++ ├── src
++     ├── lib
++     ├── commands
++     ├── logger
++ ├── tests
```

Comment:
Design question: Should testing for Bedrock templates be at the module level/ environment level or we continue to have separately? This may aid in whitelisting practices.

Reply:
We will keep current tests and look into optimizations irrespective the repo reorganization.